README.rst

```diff
@@ -59,31 +59,78 @@ it's often difficult to work with. PUDL takes the original spreadsheets, CSV fil
 and databases and turns them into a unified resource. This allows users to spend more
 time on novel analysis and less time on data preparation.
 
+PUDL is comprised of three core components:
```

Comment:
I am king of feeling like maybe we don't actually need this. And that it's more important to get strait to data access.

Reply:
IDK this section has grown on me. It might not be clear to new users that PUDL refers to code and data.

Reply:
hahah I love how we keep flip flopping on this